src/ClusterBootstrap/deploy.py

```diff
@@ -2755,14 +2780,11 @@ def kubernetes_label_nodes( verb, servicelists, force ):
                 kubernetes_label_node(cmdoptions, nodename, label+"-")
 
 
-# Label kubernete nodes with gpu types.
+# Label kubernete nodes with gpu types.skip for CPU workers
 def kubernetes_label_GpuTypes():
-    nodes = get_nodes(config["clusterId"])
-    gpuType_config = fetch_config(config, ["gpuType_config"])
-    for gpuType, nodes in gpuType_config:
-        for node in nodes:
-            nodename = kubernetes_get_node_name(node)
-            kubernetes_label_node("--overwrite", nodename, "gpuType="+gpuType)
+    for nodename,nodeInfo in config["machines"].items():
```

Comment:
What's the default value of gpu-type? What if gpu-type is not set for a node?

Reply:
Also, does this work with on-perm machine deployment?

Reply:
gpu-type default value is NULL.
I'm testing it on on-premise devbox; as for on-prem cluster, I probably need the corresponding resource to test.